src/main/webapp/dataUseLetter/DataUseLetter.js

```diff
@@ -381,7 +381,8 @@ const DataUseLetter = hh(class DataUseLetter extends Component {
       DUL.updateDUL(form).then(resp => {
         this.createRestriction();
         DUL.createDulPdf({ uid: id }).then(() => {
-          window.location.href = component.serverURL + "/dataUseLetter/show?id=" + id;
+          this.props.init(id);
+          this.props.hideSpinner();
```

Comment:
This looks unrelated to the top nav conversion. Please revert and submit as a bug fix with appropriate ticket.

Reply:
Done!